easy_infra/config.py

```diff
@@ -0,0 +1,73 @@
+import sys
```

Comment:
I broke this out to remove some cyclical imports. Nothing special happening here otherwise.